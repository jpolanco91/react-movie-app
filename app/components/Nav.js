import React from 'react';

const Nav = () => (
  <nav className="navbar is-warning is-fixed-top" role="navigation" aria-label="main navigation">
    <div className="navbar-brand">
      <a className="navbar-item" href="/">
        <img src="https://bulma.io/images/bulma-logo.png" width="112" height="28" alt="" />
      </a>
      <div
        role="button"
        className="navbar-burger burger"
        aria-label="menu"
        aria-expanded="false"
        data-target="navbarBasicExample"
      >
        <span aria-hidden="true" />
        <span aria-hidden="true" />
        <span aria-hidden="true" />
      </div>
    </div>
  </nav>
);

export default Nav;
