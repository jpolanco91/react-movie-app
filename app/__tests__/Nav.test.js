import React from 'react';
import { create } from 'react-test-renderer';
import Nav from '../components/Nav';

test('Nav snapshot test', () => {
  const c = create(<Nav />);
  expect(c.toJSON()).toMatchSnapshot();
});
