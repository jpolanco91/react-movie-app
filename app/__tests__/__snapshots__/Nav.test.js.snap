// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav snapshot test 1`] = `
<nav
  aria-label="main navigation"
  className="navbar is-warning is-fixed-top"
  role="navigation"
>
  <div
    className="navbar-brand"
  >
    <a
      className="navbar-item"
      href="https://bulma.io"
    >
      <img
        alt=""
        height="28"
        src="https://bulma.io/images/bulma-logo.png"
        width="112"
      />
    </a>
    <div
      aria-expanded="false"
      aria-label="menu"
      className="navbar-burger burger"
      data-target="navbarBasicExample"
      role="button"
    >
      <span
        aria-hidden="true"
      />
      <span
        aria-hidden="true"
      />
      <span
        aria-hidden="true"
      />
    </div>
  </div>
</nav>
`;
